src/main/java/ch/njol/skript/ScriptLoader.java

```diff
@@ -840,43 +642,34 @@ private static ScriptInfo loadScript(@Nullable Config config) {
 		
 		return scriptInfo;
 	}
-	
-	
+
 	/*
-	 * Structure loading methods
-	 */
-	/**
-	 * Loads structures of specified scripts.
-	 *
-	 * @param files the scripts to load
+	 * Script Structure Loading Methods
 	 */
-	public static List<Config> loadStructures(File[] files) {
-		Arrays.sort(files);
-		
-		List<Config> loadedFiles = new ArrayList<>(files.length);
-		for (File f : files) {
-			assert f != null : Arrays.toString(files);
-			Config config = loadStructure(f);
-			if (config != null)
-				loadedFiles.add(config);
-		}
-		
-		return loadedFiles;
-	}
 	
 	/**
-	 * Loads structures of all scripts in the given directory, or of the passed script if it's a normal file.
-	 *
-	 * @param directory a directory or a single file
-	 * @see #loadStructure(File).
+	 * Creates a script structure for every file contained within the provided directory.
+	 * If a directory is not actually provided, the file itself will be used.
+	 * @param directory The directory to create structures from.
+	 * @see ScriptLoader#loadStructure(File)
+	 * @return A list of all successfully loaded structures.
 	 */
-	public static List<Config> loadStructures(File directory) {
+	private static List<Config> loadStructures(File directory) {
```

Comment:
TP and I decided that these methods should *not* be exposed for users. I think these methods were exposed before because no proper script loading methods for File objects really existed. In fact, the only actual `loadScripts` method was for loading from the default scripts directory of the plugin. However, in an effort to remove spaghetti, proper script loading methods have been added and it is (forcibly :rofl:) recommended to use those.

Although this is a breaking change, we decided that the impact would be very low (if any impact at all).